schemas/maas-backend/tsp-auth/validate/request.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/tsp-auth/validate/request.json",
+  "description": "Request schema for tsp-auth validate. Leaving as much flexibility as possible to be resilient for different TSPs",
+  "type": "object",
+  "properties": {
+    "agencyId": {
+      "$ref": "http://maasglobal.com/core/components/common.json#/definitions/agencyId"
+    },
+    "token": {
+      "$ref": "http://maasglobal.com/core/components/common.json#/definitions/encodedQueryParam"
+    },
+    "encodedData": {
+      "$ref": "http://maasglobal.com/core/components/common.json#/definitions/encodedQueryParam"
+    },
+    "error": {
+      "$ref": "http://maasglobal.com/core/components/common.json#/definitions/errorKey"
+    },
+    "headers": {
```

Comment:
"$ref": "http://maasglobal.com/core/components/api-common.json#/definitions/headers"

Reply:
Can't use the MaaS API headers definition as this is an html endpoint, where browser is redirected to and there is no control on headers that are pushed in